core/lib/rom/relation.rb

```diff
@@ -81,6 +84,30 @@ class Relation
       schema.any?(&:read?) ? schema.to_output_hash : NOOP_OUTPUT_SCHEMA
     }
 
+    # @!attribute [r] mappers
+    #   @return [MapperRegistry] an optional mapper registry (empty by default)
+    option :mappers, default: -> { MapperRegistry.new }
+
+    # @!attribute [r] auto_struct
+    #   @return [TrueClass,FalseClass] Whether or not tuples should be auto-mapped to structs
+    #   @api private
+    option :auto_struct, reader: true, default: -> { false }
```

Comment:
reader is `:public` by default
https://github.com/dry-rb/dry-initializer/blob/0d2fa01b11a0669c61eddd0fbc4e18d7aa4094a8/lib/dry/initializer/attribute.rb#L48